docs/04-architecture/turbopack.mdx

```diff
@@ -27,3 +27,30 @@ Turbopack in Next.js requires zero-configuration for most users and can be exten
 ## Unsupported features
 
 Turbopack currently only supports `next dev` and does not support `next build`. We are currently working on support for builds as we move closer towards stability.
+
+These features are currently not supported:
+
+- `webpack()` configuration in `next.config.js`
+  - Turbopack replaces Webpack, this means that webpack configuration is not supported.
+  - For configuring Turbopack [see the documentation](/docs/app/api-reference/next-config-js/turbo).
+  - A subset of Webpack loaders [are supported](/docs/app/api-reference/next-config-js/turbo#webpack-loaders) in Turbopack.
+- Babel`(.babelrc`)
+  - Turbopack leverages the [SWC](/docs/architecture/nextjs-compiler#why-swc) compiler for all transpilation and optimizations. This means that Babel is not included by default.
+  - If you have a `.babelrc` you might no longer need it because Next.js includes common Babel plugins as SWC transforms that can be enabled. You can read more about this in [the compiler documentation](docs/architecture/nextjs-compiler#supported-features).
+  - If you still need to use Babel after verifying your particular use case is not covered, you can leverage Turbopack's [support for custom webpack loaders](/docs/app/api-reference/next-config-js/turbo#webpack-loaders) to include `babel-loader`.
+- Creating a root layout automatically in App Router.
+  - Since this behavior changes input files this is currently not supported, instead an error will be shown to add the root layout manually at the desired location.
```

Comment:
```suggestion
  - This behavior is currently not supported since it changes input files, instead, an error will be shown for you manually add a root layout in the desired location.
```